github4s/src/test/scala/github4s/integration/ReposSpec.scala

```diff
@@ -313,6 +313,57 @@ trait ReposSpec extends BaseIntegrationSpec {
     response.statusCode shouldBe notFoundStatusCode
   }
 
+  "Repos >> UserIsCollaborator" should "return true when the user is a collaborator" taggedAs Integration in {
+    val response = clientResource
+      .use { client =>
+        Github[IO](client, accessToken).repos
+          .userIsCollaborator(
+            validRepoOwner,
+            validRepoName,
+            validUsername,
+            headers = headerUserAgent
+          )
+      }
+      .unsafeRunSync()
+
+    testIsRight[Boolean](response, r => r should be(true))
+    response.statusCode shouldBe noContentStatusCode
+  }
+
+  it should "return false when the user is not a collaborator" taggedAs Integration in {
+    val response = clientResource
+      .use { client =>
+        Github[IO](client, accessToken).repos
+          .userIsCollaborator(
+            validRepoName,
```

Comment:
I think this should be `validRepoOwner`?

Reply:
I think that was meant to generate a 404.

Reply:
Doesn't the `invalidUsername` do that? We are trying to test the "correct owner, correct repo, non-collaborator user" case here.

Reply:
Yeah sorry that was meant to be `validRepoOwner`

Reply:
ah, true indeed :+1: